platform/app/prisma/migrations/20260824120000_account_credential/migration.sql

```
@@ -0,0 +1,42 @@
+    "accessTokenExpiresAt" TIMESTAMP(3),
+    "refreshTokenExpiresAt" TIMESTAMP(3),
+    "scope" TEXT,
+    "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updatedAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+
+    CONSTRAINT "AccountCredential_pkey" PRIMARY KEY ("id")
+);
+
+-- Scopes the account list and the fan-out a user delete performs.
+-- CreateIndex
+CREATE INDEX "AccountCredential_userId_idx" ON "AccountCredential"("userId");
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)